cypress/integration/models/developer/applicationinventory/application.ts

```diff
@@ -285,12 +286,23 @@ export class Application {
             });
     }
 
+    applicationDetailsTab(tab: string): void {
+        // Navigate to the application details page and click desired tab
+        this.selectApplicationRow();
+        clickByText(button, tab);
```

Comment:
clickByText(navTab, tab);

will work better here .

Reply:
Replaced it with clickByText(navTab, tab);